cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -73,7 +78,10 @@ def import_handler(request, course_key_string):
             # Do everything in a try-except block to make sure everything is properly cleaned up.
             try:
                 data_root = path(settings.GITHUB_REPO_ROOT)
-                course_subdir = "{0}-{1}-{2}".format(course_key.org, course_key.course, course_key.run)
+                if library:
+                    course_subdir = "library-v1_{0}-{1}".format(course_key.org, course_key.run)
```

Comment:
Nit: rename the variable course_subdir to make it more generic.

Is the library version number stored anywhere as a variable?